gradle.properties

```diff
@@ -1,3 +1,8 @@
 org.gradle.parallel=false
 org.gradle.caching=true
 org.gradle.configureondemand=true
+org.gradle.jvmargs=--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED \
```

Comment:
should not be submitted as part of the PR. I will update the spotless gradle plugin later today.

Reply:
All right.

Reply:
you can update your branch. Plugin has been upgraded so you don't need this anymore. Thanks for sharing!